package.json

```diff
@@ -2,14 +2,14 @@
     "name": "csharpextensions",
     "displayName": "C# Extensions",
     "description": "C# IDE Extensions for VSCode",
-    "author": "KreativJos",
```

Comment:
Would you please not change the author & publisher names, and the repository in a pull-request?